### PR TITLE
Support Android 15

### LINKED ISSUE #372: Update to Android 15 (35)
Here we are again, a new Android version appeared some time ago and is going live soon.

There are quite some changes:
- https://developer.android.com/about/versions/15/behavior-changes-15
- https://developer.android.com/about/versions/15/behavior-changes-all

We will have to take a look at what will change for this app.

### PR DESCRIPTION
**Pull request by cyb3rko**
We have a new limitation now:  
The app is not allowed to open the intent url dialog without the app being currently open anymore. I don't know if there is a solution for that, but I haven't found one so far.

Furthermore the status bar (showing the time, notification icons, ...) normally has the same color than the app bar, only on some pages it is black. I couldn't figure out yet why that is the case.

Closes #372

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,2 +1,3 @@
 [*.{kt,kts}]
```
Comment (jmattheis) [inline .editorconfig:1]: > The app is not allowed to open the intent url dialog without the app being currently open anymore. I don't know if there is a solution for that, but I haven't found one so far.

Have you enabled the "Intent Action Permission" in the settings? Or does this setting now have no effect anymore?

Comment (cyb3rko) [reply .editorconfig:1]: Hmm, testing this again and it seems to work now.  
Maybe I really just didn't toggle it last time.

**Comment by jmattheis** [comment]
FYI: I'll take some time to review this, as I currently don't have access to my android dev environment. I maybe find some time on sunday, but I'm not sure yet.

File: .editorconfig [hunk 2]
```diff
@@ -1,2 +1,3 @@
...
-ktlint_code_style = android_studio
```
Comment (jmattheis) [inline .editorconfig:2]: > Furthermore the status bar (showing the time, notification icons, ...) normally has the same color than the app bar, only on some pages it is black. I couldn't figure out yet why that is the case.

The only page is the share/push messages view right? It seems to be related to the scrollview. Wrapping the scrollview in a androidx.drawerlayout.widget.DrawerLayout like the other pages, seems to fix this.

File: app/build.gradle [hunk 3]
```diff
@@ -1,16 +1,16 @@
 plugins {
```
Comment (jmattheis) [inline app/build.gradle:1]: Could you rebase and integrate the kts changes?

**Review by jmattheis — CHANGES_REQUESTED** [review]
Tested with Pixel 8 with Android 15, looks good!

File: app/build.gradle (hunk 3 above)
Comment (cyb3rko) [reply app/build.gradle]: Whoops, wrong branch -.-

**Review by jmattheis — APPROVED** [review]

**Comment by cyb3rko** [comment]
If you found a fix for the bugged status bar, please commit that

**Pull request merged**